<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fenêtre principale</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>591</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabRecherche">
     <attribute name="title">
      <string>Recherche de client</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>561</width>
        <height>341</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayoutRecherche">
       <item row="0" column="4">
        <widget class="QLineEdit" name="leIdentifiant"/>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="lblIdentifiant">
         <property name="text">
          <string>Identifiant</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lePrenom"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblNom">
         <property name="text">
          <string>Nom</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblPrenom">
         <property name="text">
          <string>Prénom</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QDateEdit" name="deRendezvous"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="leNom"/>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="lblDaterendezvous">
         <property name="text">
          <string>Date de rendez-vous</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5" rowspan="2">
        <widget class="QPushButton" name="btnRechercherclient">
         <property name="text">
          <string>Rechercher</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="6">
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabPlanification">
     <attribute name="title">
      <string>Planification</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>28</y>
        <width>561</width>
        <height>321</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayoutPlanification">
       <item row="0" column="0">
        <widget class="QLabel" name="lblRendezvous">
         <property name="text">
          <string>Date de rendez-vous</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDateEdit" name="deRendezvousplanification"/>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btnRechercherplanification">
         <property name="text">
          <string>Rechercher</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QTableView" name="tableView_2"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>616</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <widget class="QMenu" name="menuAjouter">
     <property name="title">
      <string>Ajouter</string>
     </property>
     <addaction name="actionClient"/>
     <addaction name="actionPersonnel"/>
    </widget>
    <addaction name="menuAjouter"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionA_propos"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionClient"/>
   <addaction name="separator"/>
   <addaction name="actionPersonnel"/>
  </widget>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
  <action name="actionClient">
   <property name="icon">
    <iconset resource="logo.qrc">
     <normaloff>:/myLogo/images/logo_client.jpg</normaloff>:/myLogo/images/logo_client.jpg</iconset>
   </property>
   <property name="text">
    <string>Client</string>
   </property>
  </action>
  <action name="actionPersonnel">
   <property name="icon">
    <iconset resource="logo.qrc">
     <normaloff>:/myLogo/images/logo_personnel.png</normaloff>:/myLogo/images/logo_personnel.png</iconset>
   </property>
   <property name="text">
    <string>Personnel</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>
